### PR TITLE
Tweak documentation about exercise test suites

### PR DESCRIPTION
**Pull request by kytrinyx**
I wanted to emphasize the bit about adhering to language/framework conventions, so I pulled that out of the bulleted list.

I also tried adding the bit about randomization/skipping tests. My thinking here is that we should include anything that will let a new track maintainer on a brand new track make choices that optimize for a good user experience.

### CODE REVIEW

File: contributing-to-language-tracks/exercise-test-suites.md [hunk 1]
```diff
@@ -7,20 +7,27 @@ We simulate Test-Driven Development (TDD) by implementing the tests in order of
...
+- a manageable increase in complexity from the previous passing test
 - introduces a single new requirement
 - a minimal number of expectations required to specify the change (one expectation is preferred)
 - a readable and understandable failure message
-- adherence to the testing idioms of the language and testing framework of the track
 - does not duplicate the expectations of another test
 
+Many testing frameworks randomize the order in which tests are run. Under normal circumstances this is desireable, since it helps enforce tests that are indpendent of each other. In the context of Exercism, however, this can be confusing. When someone begins to solve an exercise, getting all the failures for all the tests in random order can make it hard to know where to begin solving the problem.
+
+In order to provide a better user experience, some language tracks only leave the first test active. All subsequent tests are then marked as "skipped" or "pending" using whatever method or directive that is provided by the testing framework in question. This means that the person solving the Exercise must then manually delete the directive from the test file each time they make a test pass.
```
Comment (ErikSchierboom) [inline contributing-to-language-tracks/exercise-test-suites.md]: Perhaps a slight rephrase to say something like: to work on the next exercise, they must manually delete the directive. I think next exercise is clearer.

Comment (kytrinyx) [reply contributing-to-language-tracks/exercise-test-suites.md]: It's all part of one single exercise, though. Let me see if I can rephrase.

(Also: Exercise shouldn't be capitalized. I'll fix that)

Comment (kytrinyx) [reply contributing-to-language-tracks/exercise-test-suites.md]: I tried this:

> In order to provide a better user experience, some language tracks only leave the first test in an exercise test suite active. All subsequent tests are marked as "skipped" or "pending" using whatever method or directive that is provided by the testing framework. This means that the person solving the exercise must manually edit the test suite to activate the tests one by one as they write their solution.

Comment (ErikSchierboom) [reply contributing-to-language-tracks/exercise-test-suites.md]: Perfect!

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**